sdk/spring/pom.xml

```diff
@@ -66,6 +66,8 @@
         <module>spring-cloud-azure-feature-management</module>
         <module>spring-cloud-azure-feature-management-web</module>
         <module>spring-cloud-azure-starter-appconfiguration-config</module>
+        <module>azure-spring-data-cosmos</module>
+        <module>azure-spring-data-cosmos-test</module>
```

Comment:
Please add the 2 modules into [spring-cloud-azure-dependencies](https://github.com/Azure/azure-sdk-for-java/blob/spring-cloud-azure-autoconfigure_4.6.0/sdk/boms/spring-cloud-azure-dependencies/pom.xml)